model/repository.js

```diff
@@ -10,6 +10,8 @@ const repositorySchema = new mongoose.Schema({
     login: String
   },
   accessToken: String,
+  buildStatus: Boolean,
+  mailService: Boolean,
```

Comment:
User should have an option to disable build status mail. It can be considered spamming.